### PR TITLE
Enabled mods outside of QuickPlay.

### PR DESCRIPTION
**Pull request by Snow1226**
I considered it.
The mod does not increase the score itself, but rather increases the restrictions, so I disabled the mod only during QuickPlay.

### CODE REVIEW

File: AlternativePlay/HarmonyPatches/MultiplayerPatch.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using HarmonyLib;
+
+namespace AlternativePlay.HarmonyPatches
+{
+    [HarmonyPatch(typeof(MultiplayerLobbyConnectionController), "HandleMultiplayerSessionManagerConnected")]
+    class MultiplayerPatch
```
Comment (Kylon99) [inline AlternativePlay/HarmonyPatches/MultiplayerPatch.cs]: Is this supposed to be an "internal" class?

File: AlternativePlay/HarmonyPatches/MultiplayerPatch.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    {
+        public static MultiplayerLobbyConnectionController.LobbyConnectionType connectionType;
+        static void Postfix(MultiplayerLobbyConnectionController __instance)
+        {
+            //AlternativePlay.Logger.Notice($"Patch MultiplayerLobbyConnenctionController {__instance.connectionState.ToString()}");
+            //AlternativePlay.Logger.Notice($"Patch MultiplayerLobbyConnenctionController {__instance.connectionType.ToString()}");
```
Comment (Kylon99) [inline AlternativePlay/HarmonyPatches/MultiplayerPatch.cs]: It's better to delete unused comments and logging.

File: AlternativePlay/HarmonyPatches/MultiplayerPatch.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿using HarmonyLib;
+
+namespace AlternativePlay.HarmonyPatches
+{
+    [HarmonyPatch(typeof(MultiplayerLobbyConnectionController), "HandleMultiplayerSessionManagerConnected")]
+    class MultiplayerPatch
+    {
+        public static MultiplayerLobbyConnectionController.LobbyConnectionType connectionType;
+        static void Postfix(MultiplayerLobbyConnectionController __instance)
```
Comment (Kylon99) [inline AlternativePlay/HarmonyPatches/MultiplayerPatch.cs]: Is this a "private" method?

**Review by Kylon99 — CHANGES_REQUESTED** [review]
Just some clean up is required.

**Comment by Snow1226** [comment]
Sorry, it was omitted from the description, and has been corrected.

**Pull request merged**